src/Nethermind/Nethermind.Specs/SepoliaSpecProvider.cs

```diff
@@ -29,7 +29,7 @@ public void UpdateMergeTransitionInfo(long? blockNumber, UInt256? terminalTotalD
             TerminalTotalDifficulty = terminalTotalDifficulty;
     }
 
-    public ulong NetworkId => Core.BlockchainIds.Rinkeby;
+    public ulong NetworkId => Core.BlockchainIds.Sepolia;
```

Comment:
This could be problematic: since `SepoliaSpecProvider` has been introduced, it has used `BlockchainIds.Rinkeby` (4) as `NetworkId` instead of the expected `BlockchainIds.Sepolia` (11155111).

What should we do with this? Do we keep the old `4` (as a hard-coded value) or do we change it to use expected `BlockchainIds.Sepolia`

Reply:
Not a problem, we load it from chainspec in normal operation - this could be considered bugfix.